common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4886,11 +4886,6 @@
           "Catalog <catalogName> does not support <operation>."
         ]
       },
-      "COLLATION" : {
-        "message" : [
-          "Collation is not yet supported."
-        ]
-      },
```

Comment:
why are we removing this now?

Reply:
UNSUPPORTED_FEATURE.TRIM_COLLATION was removed and this is the leftover that doesn't make sense without it.